Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV GO111MODULE=on
 RUN apk add curl git build-base
 
 # Install linter
-RUN curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b $HOME/bin v1.44.0
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $HOME/bin v1.44.0
```

Comment:
I had a warning during docker build:
```
golangci/golangci-lint err this script is deprecated, please do not use it anymore. check https://github.com/goreleaser/godownloader/issues/207
```
The replacement was taken from the [official manual](https://golangci-lint.run/usage/install/#linux-and-windows).